src/Tizen.NUI/src/internal/Interop/Interop.FrameBroker.cs

```diff
@@ -0,0 +1,137 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Runtime.InteropServices;
+using Tizen.Applications;
+using Tizen.NUI;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class FrameBroker
+        {
+            internal enum ErrorCode
+            {
+                None = Tizen.Internals.Errors.ErrorCode.None,
+                InvalidParameter = Tizen.Internals.Errors.ErrorCode.InvalidParameter,
+                OutOfMemory = Tizen.Internals.Errors.ErrorCode.OutOfMemory,
+                AppNotFound = -0x01100000 | 0x21,
```

Comment:
it looks different from other enum. (other enums have meaningful names) the raw hex value seems not good. is it possible to replace hex value as some named variables like others?

Reply:
Ok, fixed.